index.js

```diff
@@ -31,6 +35,7 @@ var ERROR_TYPES = {
  * config.enable_debug : true|false
  * config.logger: <pass in a logger that has debug() and error() functions>
  * config.cache: { module: <redis|node-cache>, options: <cache-service-options> }
+ * config.cert_background_refresh: true|false
```

Comment:
Why would you want this to be an option? I can't think of a reason why you wouldn't want the background refresh.